frontend/simple/views/Invite.vue

```diff
@@ -164,21 +272,73 @@ export default {
             },
             mailbox
           )
-          await backend.publishLogEntry(member.state.attributes.mailbox, invite)
-
-          // We need to make a record of the invitation in the group's contract
-          const latest = await backend.latestHash(this.$store.state.currentGroupId)
-          const invited = new Events.HashableGroupRecordInvitation(
-            {
-              username: member.state.attributes.name,
-              inviteHash: invite.toHash(),
-              sentDate
-            },
-            latest
-          )
-          await backend.publishLogEntry(this.$store.state.currentGroupId, invited)
+          const state = await latestContractState(member.contractId)
+          // We need to determine if the invitation needs an invite
```

Comment:
it's not clear to me what this is saying, how is it that an invitation needs an invite? can this comment be expended to make it clearer what's going on here?